tests/models/test_ac_link.py

```diff
@@ -0,0 +1,297 @@
+    )
+
+
+def test_parallel_ac_links(ac_lib: Library):
+    """
+    The network has 2 AC nodes connected by 2 parallel links,
+    where reactance is 1 for line L1, and 2 for line L2.
+    We expect flow to be te twice bigger on L1 than on L2.
+
+    Node 1 carries the demand of 100 MW,
+    node 2 carries the generator with a cost of 35 per MWh.
+
+    We check that final cost matches the demand: 100 * 35 = 3500,
+    and that flow on L1 is -66. MW while flow on L2 is only -33.3 MW.
+    """
+    ac_node_model = ac_lib.models["ac-node"]
+    ac_link_model = ac_lib.models["ac-link"]
+
+    database = DataBase()
+    database.add_data("D", "demand", ConstantData(100))
+
+    database.add_data("G", "p_max", ConstantData(100))
+    database.add_data("G", "cost", ConstantData(35))
+
+    database.add_data("L1", "reactance", ConstantData(1))
+    database.add_data("L2", "reactance", ConstantData(2))
+
+    node1 = Node(model=ac_node_model, id="1")
+    node2 = Node(model=ac_node_model, id="2")
+    demand = create_component(
+        model=DEMAND_MODEL,
+        id="D",
+    )
+    gen = create_component(
+        model=GENERATOR_MODEL,
+        id="G",
+    )
+    link1 = create_component(
+        model=ac_link_model,
+        id="L1",
+    )
+    link2 = create_component(
+        model=ac_link_model,
+        id="L2",
+    )
+
+    network = Network("test")
+    network.add_node(node1)
+    network.add_node(node2)
+    network.add_component(demand)
+    network.add_component(gen)
+    network.add_component(link1)
+    network.add_component(link2)
+    network.connect(PortRef(demand, "balance_port"), PortRef(node1, "injections"))
+    network.connect(PortRef(gen, "balance_port"), PortRef(node2, "injections"))
+    network.connect(PortRef(link1, "port1"), PortRef(node1, "links"))
+    network.connect(PortRef(link1, "port2"), PortRef(node2, "links"))
+    network.connect(PortRef(link2, "port1"), PortRef(node1, "links"))
+    network.connect(PortRef(link2, "port2"), PortRef(node2, "links"))
+
+    scenarios = 1
+    problem = build_problem(network, database, TimeBlock(1, [0]), scenarios)
+    status = problem.solver.Solve()
+
+    assert status == problem.solver.OPTIMAL
+    assert problem.solver.Objective().Value() == pytest.approx(3500, abs=0.01)
+
+    assert OutputValues(problem).component("L1").var("flow").value == pytest.approx(
+        -66.67, abs=0.01
+    )
+    assert OutputValues(problem).component("L2").var("flow").value == pytest.approx(
+        -33.33, abs=0.01
+    )
+
+
+def test_parallel_ac_links_with_pst(ac_lib: Library):
+    """
+    Same case as in parallel_ac_links but:
+     - flow is restricted to 50 MW on line L1, so it cannot
+       anymore transfer 66,7MW
+     - flow can be influenced by a phase shifter on line L2
+
+    We expect the case to be feasible thanks to the phase shifter,
+    which will allow to balance the flow between the 2 lines.
+    Therefore we expect flows to be 50 MW on both lines.
```

Comment:
Note: actually there is probably an infinite set of solutions here, since there is no cost associated to the phase shifting.

Reply:
Then is it worth to enforce unicity of a solution at least for the test to be robust to a change of solver for instance ?